src/styles/blocks/cards.scss

```
@@ -0,0 +1,107 @@
+.cards {
+  width: maxWidth($card-size,$margin-size);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

```suggestion
  justify-content: center;
```
we have $margin-size for spacing